ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -48,15 +48,16 @@
 #endif
 
 static const gchar *tflite_accl_support[] = {
-  ACCL_AUTO_STR,
-  ACCL_DEFAULT_STR,
   ACCL_CPU_NEON_STR,
   ACCL_CPU_STR,
   ACCL_GPU_STR,
   ACCL_NPU_STR,
   NULL
 };
 
+static const gchar *tflite_accl_auto = ACCL_CPU_NEON_STR;
```

Comment:
You may need to determine this based on the given architecture. x64 is not to be ignored :)

Reply:
aah yes, that will be good. I will update as per this.

Reply:
Uh __x86_64__ does not have NEON. Do you intend to designate SSE/AVS w/ NEON?

Reply:
Yes, I was intending NEON was for generic SIMD based processing.
How about rename CPU_NEON -> CPU_SIMD?

Reply:
For string values (```ACCL_*_STR```), we can add ```"cpu.simd"```
For enum values, let's define ```ACCL_CPU_SIMD = ACCL_CPU_NEON```.
For enum in API, we can define ```ML_NNFW_HW_CPU_SIMD = ML_NNFW_HW_CPU_NEON``` to keep the backward compatibility and minimize API changes, making it easier to proceed with ACR.

Reply:
Added it in #2377 as this PR is already touching too many files.